INSTALLATION.md

```diff
@@ -24,51 +24,23 @@ git clone git@github.com:1024pix/pix.git
 cd pix
 ```
```

Comment:
J'ai cloné le repo en local avec un autre profil GitHub pour vérifier.
Les containers locaux déjà démarrés font échouer l'installation
`ERROR: for pix_postgres_1  Cannot start service postgres: driver failed programming external connectivity on endpoint pix_postgres_1 (9e77b7be2593c0d327bCreating pix_redis_1    ... error
WARNING: Host is already in use by another container
`

Je propose d'ajouter dans les pré-requis de dire que le port 5432 ne doit pas être pris (ou de modifier le fichier blabla)

Reply:
On ne trouve pas de commande pour tester qui soit suffisamment portable (Ubuntu, Mac). On se contente d'une ligne dans le fichier INSTALLATION.md